### PR TITLE
[metric-example/query-by-example.md] Removed optional parameters from :last scenario

### PR DESCRIPTION
**Pull request by osl-nor**
Removed optional parameters from :last example.

### CODE REVIEW

**Comment by myieye** [comment]
Unless you've already reached out to someone else to merge this, it would be nice to know if you're expecting this to just get merged or are waiting for an approval from someone more familiar with the content.

**Review by tobbaumgartner — APPROVED** [review]

**Comment by osl-nor** [comment]
Thanks, was waiting for Tobias to have a look. 
Could be merged now that he has approved it.

**Pull request merged**
